WcaOnRails/app/models/competition.rb

```diff
@@ -846,9 +846,11 @@ def self.search(query, params: {}, managed_by_user: nil)
   # See https://github.com/thewca/worldcubeassociation.org/wiki/wcif
   def to_wcif
     managers = self.managers
+    competitor_id = 0
     persons_wcif = registrations.map do |r|
       managers.delete(r.user)
-      r.user.to_wcif(self, r.to_wcif)
+      competitor_id += 1
```

Comment:
You could've also used `registrations.map.with_index do |r, competitor_id| ...`.

Reply:
awesome, thanks!